src/pages/DataSubmissionForm.js

```diff
@@ -41,7 +42,8 @@ export const DataSubmissionForm = () => {
     ]),
 
     form({}, [
-      h(DataSubmissionStudyInformation, { onChange })
+      h(DataSubmissionStudyInformation, { onChange }),
+      h(DataSubmissionNihAnvilUse, { onChange, formData })
```

Comment:
The study information component doesn't take in any existing formData - should it? (is the purpose to set values for a saved submission form?)

Reply:
Thanks for the feedback!

You can ignore the formData prop for now! 
I was trying to work out some bugs with my ticket and that was left over from my experimentation. I think I found a workaround, but let me know if you have ideas on a better approach (I'll comment closer to the actual code to provide context)